carbon/app.py

```diff
@@ -67,7 +67,7 @@ def people():
         .where(persons.c.APPOINTMENT_END_DATE >= datetime(2009, 1, 1)) \
         .where(func.upper(dlcs.c.ORG_HIER_SCHOOL_AREA_NAME).in_(AREAS)) \
         .where(persons.c.PERSONNEL_SUBAREA_CODE.in_(PS_CODES)) \
-        .where(func.upper(persons.c.JOB_TITLE).in_(TITLES))
+        .where(func.upper(persons.c.JOB_TITLE).in_(TITLES))  # noqa: E711
```

Comment:
Unfortunately, this is required due to how SQLAlchemy is handling the operator overloading for things like `some_column != None`. pep8 says the comparison should be `some_column is not None` but this will not generate valid SQL.